cmd/eventprocessor.go

```diff
@@ -123,27 +142,45 @@ func main() {
 		metricsSink.IncrCounter(messagesReceived, 1)
 
 		message := *messages[0]
-		log.Debugf("Message: %v", message)
 
+		loggingFields := log.Fields{"msg id": *message.MessageId}
```

Comment:
app and groups have used 'MsgId' - might be good to keep these consistent for queries later.